src/outputs/outputs_main.py

```diff
@@ -119,11 +120,12 @@ def run_outputs(  # noqa: C901
     # Running Intram by PG output (GB)
     if config["global"]["output_intram_by_pg_gb"]:
         OutputMainLogger.info("Starting Intram by PG (GB) output...")
-        output_intram_by_pg(
+        intram_tot_dict = output_intram_by_pg(
             outputs_df,
             ni_full_responses,
             pg_detailed,
             config,
+            intram_tot_dict,
             write_csv,
             run_id,
             uk_output=False,
```

Comment:
I can't comment on the specific line, but intram_tot_dict is missing from from the PG output args (line 148)

Reply:
the GB will be a different total to the UK, so I thought I'd leave it out. What do you think?

Reply:
Yes, let's ask Dan (it'll need taking out of output_intram_by_pg if not needed)

Reply:
sorry, I didn't think about the errors.